### ISSUE TITLE
Validation of sensing period

### ISSUE THREAD

**Issue by EmanuelCastanho** [SDFIdk/CDSETool#24]
## Describe the bug
When `startDate >= completionDate` it returns the error `IndexError: pop from empty list`.

### Steps to reproduce
`"startDate": '2023-11-16', "completionDate": '2023-11-15'` or `"startDate": '2023-11-16', "completionDate": '2023-11-16'`

### Expected behavior
I would suggest the following:
- When `startDate == completionDate`, `completionDate=completionDate+ timedelta(days=1)`
- When `startDate > completionDate`, error message stating that startDate should be smaller than completionDate

### Environment
 - OS: MacOS

**Comment by subnet001** [comment]
I agree, program should fail faster if no query results are available. Date validation should be a separate step i think.
